python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -463,28 +463,47 @@ def run_transformer_int4_gpu(repo_id,
             in_out_len = in_out.split("-")
             in_len = int(in_out_len[0])
             out_len = int(in_out_len[1])
-            # As different tokenizer has different encodings,
-            # in_len.txt maybe shorter than we need,
-            # use much longer context to make sure input length
-            test_length = min(in_len*2, 8192)
-            while test_length not in [32, 256, 1024, 2048, 8192] and test_length < 8192:
-                test_length = test_length * 2
-            # For the sequence length not in [32, 256, 1024, 2048, 8192], it will be truncated from 8192.txt.
-            test_length = min(test_length, 8192)
-            input_str = open(f"prompt/{test_length}.txt", 'r').read()
-            if lookahead:
-                question = "Can you please summarize this article?"
-                question_tokens = tokenizer.encode(question, return_tensors="pt")
-                max_article_len = in_len - question_tokens.size(1)
-                article_ids = tokenizer.encode(input_str, return_tensors="pt")
-                if article_ids.size(1) > max_article_len:
-                    article_ids = article_ids[:, :max_article_len]
-                input_ids = torch.cat((article_ids, question_tokens), dim=1)
-            else:
+            if not lookahead or conf['task'] == 'continuation':
+                # As different tokenizer has different encodings,
+                # in_len.txt maybe shorter than we need,
+                # use much longer context to make sure input length
+                test_length = min(in_len*2, 8192)
+                while test_length not in [32, 256, 1024, 2048, 8192] and test_length < 8192:
+                    test_length = test_length * 2
+                # For the sequence length not in [32, 256, 1024, 2048, 8192], it will be truncated from 8192.txt.
+                test_length = min(test_length, 8192)
+                input_str = open(f"prompt/continuation/{test_length}.txt", 'r').read()
                 # As different tokenizer has different encodings,
                 # slice the input_ids to ensure the prompt length is required length.
                 input_ids = tokenizer.encode(input_str, return_tensors="pt")
                 input_ids = input_ids[:, :in_len]
+            elif conf['task'] == 'summarize':
+                if in_len == 512:
```

Comment:
Maybe add a map & deal with the in_len not in [512, 1024, 2048] case.

Reply:
Add the short prompt cnn-64.

Reply:
Thanks. I have added these cases.